components/StarkNet/modules/starknet_versions/pages/version_notes.adoc

```diff
@@ -5,10 +5,17 @@ The following version notes cover the ongoing version changes to StarkNet. To ge
 
 == StarkNet Environments
 
-Within StarkNet's deployment pipeline there are separate and distinct networks that operate independently of each other for testing before deployment. See https://docs.starknet.io/documentation/useful_info/[here] for more information. 
+Within StarkNet's deployment pipeline there are separate and distinct networks that operate independently of each other for testing before deployment. See https://docs.starknet.io/documentation/useful_info/[here] for more information.
```

Comment:
Rephrase this in terms of the user. Use second person.

Style guideline: Use this phrasing: For more information, see <topic_title>.